### ISSUE TITLE
add 'accept terms'

### ISSUE THREAD

**Issue by sparrell** [sparrell/SbomPoc-sFractal#3]
there should be 3 routes 
- / status (home page), 
- /phoenix (already exists at default which needs to change), 
- /dashboard (already exists),
- /update

The restrictions on the pages are:
- /dashboard should only be accessible if authenticated (authentication is separate issue), and user is on dashboard list of authorized users
- /update should only be accessible if authenticated (authentication is separate issue), and user is on dashboard list of authorized users, and then user can only update status on their organization, not everyone else's. Admin users can update any organization
- /phoenix and /status pages content is "warning ... (Duncan to supply text) until they click acknowledge button. Ie anyone can see the real content of these pages, but only after clicking acknowledge
- /status page is a series of tabs:
   - overall
   - project
   - organization
   - interval
   - sprint
   - opensource
- each of status tabs will consist of some explanation text and some data graphically
- each status tab will be a summary and can be clicked on to drill down. Eg org page is status of all tasks across all orgs (maybe color coded by org) and clicking on an org brings you to a view of just that org's status.
- status is a result of data in a db that gets administered via update page.
- there are 10 orgs (https://github.com/oasis-tcs/openc2-usecases/tree/master/SBOM-PoC#2---organizations-participating) which will grow to appx 50
- there are 6 projects (OpenC2, SBOM, IACD, CACAO, OCA, SCAP2) which may grow or shrink
- there are 3 intervals (single, multi, mega)
- there are N sprints which are two week intervals June 4-18, June18-Jul2, Jul 2-16, Jul 16-30, .... late October
- there are N open source projects involved - will need to list and admins (of any org) will be able to add more
- org admins enter entries (which can be tasks or goals) for that organization. They may have zero-to-all projects associated with a given entry. They should have one of single/multi/mega. They may have 0, 1, ..N open source repos associated with a task
- an entry starts out unassociated with any sprint and is considered 'not started' from sprint viewpoint. As the task/goal is started to be worked on, it becomes 'being worked' for the sprint that it starts. An admin can mark it 'complete' for a given sprint. being worked and complete can only be marked on sprints prior to and including the current sprint (eg saying complete next sprint is not allowed)
- status consists of a view of the not started, being worked, complete sliced/diced depending on the view

**Comment by sparrell** [comment]
start simpler. No database. No users/authorization. All pages static and 'tabs' above are pages linked off a home page.
Ideally pages would be markdown, not html, so users could to PR's to update
